### ISSUE TITLE
visual genome dataset

### ISSUE THREAD

**Issue by liyangliu** [jwyang/faster-rcnn.pytorch#61]
@jwyang, I noticed you are doing experiments on visual genome dataset. May I ask how you split the dataset for training and validation as there is severe class imbalance?

**Comment by jwyang** [comment]
Hi, @liyangliu 

there is no standard split on this dataset. Normally, we use a ratio 7:1:2. Yes, there is severe class imbalance. I used the same way as [Perter's code](https://github.com/peteanderson80/bottom-up-attention). The class imbalance is okay from what I see. Since the class distribution of test set is the same to the training set.

**Comment by akira-l** [comment]
they use a split method as vg_1600-400-20 in [https://github.com/peteanderson80/bottom-up-attention](url) but in this project, the code use vg_150-50-50. It seems that the model provided can not be used directly
